server/src/models/results/populate.js

```diff
@@ -30,7 +27,7 @@ export default async function({models}){
       "planned_spend_pa_last_year",
       "planned_fte_pa_last_year",
     ], key => {
-      sub_program[key] = null_preserving_to_number(sub_program[key]);
+      sub_program[key] = _.isNaN(sub_program[key]) ? null : +sub_program[key];
```

Comment:
Oh, wait, there's a missing conversion to number here, should be `_.isNaN(+sub_program[key])`.

Reply:
Oh hmm this was the original code but yeah I think you're right